docs/cli.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Getting Started with the CLI
+---
+
+import CLITabs from '@theme/CLITabs';
+import TabItem from '@theme/TabItem';
+
+The [Svix CLI](https://github.com/svix/svix-cli) is one of the fastest ways to interact with our API.
+
+With the Svix CLI you can:
+ - Interact with our API
+ - Validate Webhook payloads
+ - Quickly open our docs
+
+In this tutorial, we will learn how to install, update, and use the Svix CLI.
```

Comment:
They don't care about updating. They haven't fallen in love with the CLI!
The goal with this tutorial is to make them fall in love!